examples/AnnotatorForClinicalDataExample.java

```diff
@@ -0,0 +1,543 @@
+import com.ibm.watson.health.acd.v1.model.DeleteProfileOptions;
+import com.ibm.watson.health.acd.v1.model.DeleteUserSpecificArtifactsOptions;
+import com.ibm.watson.health.acd.v1.model.DeployCartridgeOptions;
+import com.ibm.watson.health.acd.v1.model.DeployCartridgeResponse;
+import com.ibm.watson.health.acd.v1.model.Flow;
+import com.ibm.watson.health.acd.v1.model.FlowEntry;
+import com.ibm.watson.health.acd.v1.model.GetAnnotatorsByIdOptions;
+import com.ibm.watson.health.acd.v1.model.GetAnnotatorsOptions;
+import com.ibm.watson.health.acd.v1.model.GetFlowsByIdOptions;
+import com.ibm.watson.health.acd.v1.model.GetFlowsOptions;
+import com.ibm.watson.health.acd.v1.model.GetProfileOptions;
+import com.ibm.watson.health.acd.v1.model.GetProfilesOptions;
+import com.ibm.watson.health.acd.v1.model.ListStringWrapper;
+import com.ibm.watson.health.acd.v1.model.ServiceApiBean;
+import com.ibm.watson.health.acd.v1.model.ServiceStatus;
+import com.ibm.watson.health.acd.v1.model.UpdateFlowsOptions;
+import com.ibm.watson.health.acd.v1.model.UpdateProfileOptions;
+import com.ibm.watson.health.acd.v1.util.FlowUtil;
+
+/**
+ * Sample ACD Java SDK Calls
+ */
+public class AnnotatorForClinicalDataExample {
+
+	public static String ACD_APIKEY="<YOUR_APIKEY>";
+	public static String ACD_URL="https://us-east.wh-acd.cloud.ibm.com/wh-acd/api";
+	//public static String ACD_URL="https://us-south.wh-acd.cloud.ibm.com/wh-acd/api";
+	public static String version = "2020-06-01";
+	public static String name="AnnotatorForClinicalData";
+	public static AnnotatorForClinicalData acd = null;
+
+	static {
+
+		Authenticator authenticator = new BasicAuthenticator("apikey", ACD_APIKEY);
+		acd = new AnnotatorForClinicalData(
+				version,
+				AnnotatorForClinicalData.DEFAULT_SERVICE_NAME,
+				authenticator);
+		acd.setServiceUrl(ACD_URL);
+
+	}
+
+	public static void main(String[] args) {
+
+		/* Analyze */
+		// analyze();
+		// analyzeWithFlow();
+
+		/* Cartridge deployment */
+		// getCartridges();
+		// getCartridge();
+		// deployCartridge();
+		// redeployCartridge();
+		// legacyDeploy();
+
+		/* Profile crud operations */
+		// createProfile();
+		// getProfiles();
+		// getProfile();
+		// updateProfile();
+		// deleteProfile();
+
+		/* Annotator flow crud operations */
+		// createFlow();
+		// getFlows();
+		// getFlow();
+		// updateFlow();
+		// deleteFlow();
+
+		/* Misc */
+		// deleteTenantData();
+		// getHealthCheck();
+		// getAnnotators();
+		//getAnnotator();
+
+	}
+
+	/*
+	 * Annotator for Clinical Data Text Analysis Calls
+	 */
+
+	// POST /analyze
+	public static void analyze() {
+
+		List<String> annotators = Arrays.asList(
+				Name.CONCEPT_DETECTION,
+				Name.NEGATION);
+
+		Flow flow = new FlowUtil.Builder()
+				.annotators(annotators).build();
+
+		String text = "Patient has lung cancer, but did not smoke. She may consider chemotherapy as part of a treatment plan.";
+
+		ContainerGroup response = acd.analyze(text, flow);
```

Comment:
Should these examples demonstrate including a try-catch in the methods?  Currently only the 'deploy' methods show this?

Reply:
I only added try/catches where exceptions were explicitly required to be handled.